tests/test_tls.py

```diff
@@ -0,0 +1,35 @@
+                "keypassword": "deleteme",
+                "keystore_password": "deleteme"
+            }
+        }
+    }
+    try:
+        utils.upload_dcos_test_jar()
+        utils.require_spark(additional_options=service_options)
+        yield
+    finally:
+        utils.teardown_spark()
+
+
+@pytest.mark.sanity
+def test_driver_over_tls():
```

Comment:
Hey @farhan5900 sharing what I wrote on Slack but when I ran the CI build this test worked (yay!) but when I took a look at the args submitted, it included 
```
> --conf spark.ssl.noCertVerification=true
```
that doesn't seem right? see [build](https://teamcity.mesosphere.io/viewLog.html?tab=buildLog&logTab=tree&filter=debug&expand=all&buildId=1979359&_focus=140014)

Reply:
I have fixed it by providing SSL configurations while submitting driver to the dispatcher. Also moved keystore to the secret store.